cvat/apps/quality_control/serializers.py

```diff
@@ -0,0 +1,144 @@
+
+            'low_overlap_threshold',
+
+            'oriented_lines',
+            'line_orientation_threshold',
+
+            'compare_groups',
+            'group_match_threshold',
+
+            'check_covered_annotations',
+            'object_visibility_threshold',
+
+            'panoptic_comparison',
+
+            'compare_attributes',
+        )
+        read_only_fields = ('id', 'task_id', )
+
+        extra_kwargs = {
+            k: {'required': False}
+            for k in fields
+        }
+
+        for field_name, help_text in {
+            'iou_threshold': "Used for distinction between matched / unmatched shapes",
+            'low_overlap_threshold': (
+                "Used for distinction between strong / weak (low_overlap) matches"
+            ),
+            'oks_sigma': """
+                Like IoU threshold, but for points.
+                The percent of the bbox area, used as the radius of the circle around the GT point,
+                where the checked point is expected to be.
+                Read more: https://cocodataset.org/#keypoints-eval
+            """,
+            'line_thickness': """
+                Thickness of polylines, relatively to the (image area) ^ 0.5.
+                The distance to the boundary around the GT line,
+                inside of which the checked line points should be
+            """,
+            'oriented_lines': "Indicates that polylines have direction",
+            'line_orientation_threshold': """
+                The minimal gain in the GT IoU between the given and reversed line directions
+                to consider the line inverted. Only useful with the 'oriented_lines' parameter
+            """,
+            'compare_groups': "Enables or disables annotation group checks",
+            'group_match_threshold': """
+                Minimal IoU for groups to be considered matching,
+                used when the 'compare_groups' is enabled
+            """,
+            'check_covered_annotations': (
+                "Check for partially-covered annotations, useful in segmentation tasks"
+            ),
+            'object_visibility_threshold': """
+                Minimal visible area percent of the spatial annotations (polygons, masks)
+                for reporting covered annotations,
+                useful with the 'check_covered_annotations' option
+            """,
+            'panoptic_comparison': (
+                "Use only the visible part of the masks and polygons in comparisons"
+            )
+        }.items():
+            extra_kwargs.setdefault(field_name, {}).setdefault(
+                'help_text', textwrap.dedent(help_text.lstrip('\n'))
+            )
+
+    def validate(self, attrs):
+        for k, v in attrs.items():
+            if k.endswith('_threshold') or k in ['oks_sigma', 'line_thickness']:
+                if not 0 <= v <= 1:
+                    raise serializers.ValidationError(f"{k} must be in the range [0; 1]")
```

Comment:
I can add such type, but I don't see how this resolves the problem with forgetting, since the serializer fields are mostly added in the meta. The current solution checks for all `*_threshold` fields, so new fields will automatically be covered, if the follow the naming.